src/BuildScriptGenerator/Node/NodeScriptGeneratorOptions.cs

```diff
@@ -18,5 +18,7 @@ public class NodeScriptGeneratorOptions
         public string NpmRegistryUrl { get; set; }
 
         public string NodeVersion { get; set; }
+
+        public bool EnableNodeMonorepoBuild { get; set; }
```

Comment:
repo => Repo ?

Reply:
_monorepo_ is a word, so i think keep it as same will be good.